### DISCUSSION TITLE
Se busca integrantes para equipo

### DISCUSSION THREAD

**Discussion by DavidSaboredo** [Launch-X-Latam/MisionFrontEnd#581] [Team Posts]
Hola que tal nos faltan integrantes manden lx06085@innovaccion.mx 
desde ya saludos.

**Comment by gapman20** [comment]
Hola tambien ando buscando equipo 
aun tienes espacio?

**Comment by DavidSaboredo** [comment]
Hola que tal si pasame tu correo de launch x y armamos un grupo ahi

**Comment by JOSHU411** [comment]
Tienen espacio para otro? He checado con otros pero no contestan y ya me asusté jeje

**Comment by gapman20** [comment]
Mi correo es lx16569@innovaccion.mx

**Comment by JERT1995** [comment]
Buen día, si tienes espacio te agradecería agregarme: lx16122@innovaccion.mx

**Comment by ALISONQS** [comment]
Hola c: ¿todavía les falta alguien? este es mi correo: lx06210@innovaccion.mx seria un gusto hacer equipo.

**Comment by DeusGarrix4k** [comment]
Yo quiero tambien quisiera estar en tu equipo lx15821@innovaccion.mx

**Comment by ErikKN1** [comment]
yo tambien estoy en busca de equipo, ¿me puedo integrar?

**Comment by JERT1995** [comment]
Es un equipo en trams no? Acabo de crear uno, amigos que no pusieron su correo envíelo, y si alguien nos orienta que sigue seria fabuloso

**Comment by JERT1995** [comment]
Teams *

**Comment by dycalixto** [comment]
Hola, ¿aún hay espacio para alguien más? Si es así, este es mi correo lx16721@innovaccion.mx

**Comment by LuisSerranoGtz** [comment]
Hola, todavia tienen lugar? lx02079@innovaccion.mx

**Comment by alexrdzuwu** [comment]
un hay lugar?
lx04099@innovaccion.mx

**Comment by children01** [comment]
algun lugar ?
lx10732@innovaccion.mx

**Comment by Murruco** [comment]
Hay espacio

**Comment by AnghelinaAlva** [comment]
lx13309@[innovaccion.mx
Todavía necesitan intergrante?

**Comment by dansuca** [comment]
Como estan algún equipo disponible ?  lx10148@innovaccion.mx

**Comment by jlnunezobando** [comment]
Busco equipo lx08651@innovaccion.mx, por favor si me incluyen.

**Comment by joseandresgavilanes** [comment]
Busco equipo  ***@***.***

**Comment by AlexGeovanni** [comment]
yo igual necesito un equipo por si alguien tiene un espacio para mi

**Comment by Fcovaldz** [comment]
yo tmbn busco equipo para el Hackaton mi correo es: lx00111@innovaccion.mx
